backend/app/main.py

```diff
@@ -2,25 +2,38 @@
 
 from app.api import api
 from app.core.app_state import app_state
-from app.matrix.background_validator import background_validator
+from app.matrix.background_validator import BackgroundValidater
 
 # from app.core.background_runner import matrix_bot_runner
 
 app = FastAPI(docs_url="/api/docs", redoc_url="/api/redoc", openapi_url="/api/openapi")
 
+background_validator = None
+
 
 @app.on_event("startup")
 async def app_startup():
     # Fetch the initial sync from matrix homeserver
-    await app_state.matrix_bot_runner.initialise_bot()
+    global background_validator
```

Comment:
This is ugly, I think it would be better to put it in app state if we really need to do this.

Reply:
Or maybe app_state could hold a list of running background tasks?

Reply:
Ok, I'll see what I can do for this.